tests/unit/src/test/resources/mtags-scala3/example/ToplevelImplicit.scala

```diff
@@ -0,0 +1,10 @@
+package example
+
+// This wasn't possible in Scala 2
+implicit class Xtension/*example.Xtension().*//*example.Xtension#*/(number/*example.Xtension#number.*/: Int) {
```

Comment:
I see it's been like this all along but why do we emit a method for implicit class?